packages/components/src/components/Dropdown/Dropdown.tsx

```diff
@@ -47,12 +50,22 @@ export const DropdownTrigger = (
 };
 DropdownTrigger.displayName = 'Dropdown.Trigger';
 
-export type DropdownContentProps = { children?: ReactNode; 'data-test-id'?: string };
+export type DropdownContentProps = {
+    children?: ReactNode;
+    'data-test-id'?: string;
+    onOpen?: () => void;
+    onClose?: () => void;
+};
 
 export const DropdownContent = (
-    { children, 'data-test-id': dataTestId = 'fondue-dropdown-content' }: DropdownContentProps,
+    { onOpen, onClose, children, 'data-test-id': dataTestId = 'fondue-dropdown-content' }: DropdownContentProps,
     ref: ForwardedRef<HTMLDivElement>,
 ) => {
+    const localRef = useRef(null);
+    const dropdownIsOpen = useRef(false);
+
```

Comment:
unneccessary linebreak?

Reply:
just separating react hooks from custom hooks, idk why I do it 🤣